src/types/nitro.ts

```diff
@@ -278,7 +281,14 @@ export interface NitroOptions extends PresetOptions {
   wasm?: UnwasmPluginOptions;
   experimental?: {
     legacyExternals?: boolean;
-    openAPI?: boolean;
+    openAPI?:
```

Comment:
I think we can start adding `openapi` top level config key preparing for 3.10 (next month release) 👍🏼 

So experimental is a boolean and top level openapi is object

Reply:
That makes so much sense

Reply:
I’ve added this. Now you have to pass `experimental.openAPI: true` to activate the feature, and then just `openAPI: { … }` to configure. 👍